cli/src/trusted_operation.rs

```diff
@@ -133,14 +133,8 @@ fn send_request(
 	_chain_api.subscribe_events(events_in).unwrap();
 
 	loop {
-		let ret: ProcessedParentchainBlockArgs = _chain_api
-			.wait_for_event::<ProcessedParentchainBlockArgs>(
-				TEEREX,
-				"ProcessedParentchainBlock",
```

Comment:
@BillyWooo these two are the values you can fill in for the static even implementation.